include/swift/AST/DiagnosticsSema.def

```diff
@@ -3034,6 +3032,11 @@ ERROR(autodiff_attr_original_decl_none_valid_found,none,
       "could not find function %0 with expected type %1", (DeclNameRef, Type))
 ERROR(autodiff_attr_original_decl_not_same_type_context,none,
       "%0 is not defined in the current type context", (DeclNameRef))
+ERROR(autodiff_attr_original_void_result,none,
+      "cannot differentiate void function %0", (DeclName))
+ERROR(autodiff_attr_original_multiple_semantic_results,none,
+      "cannot yet differentiate functions with more than one semantic result "
```

Comment:
- "Semantic result" isn't really a user-facing term until we added the whole "inout differentiation" chapter to the manifesto (and the language book in the future)... Something more specific to `inout` would be better.
    > cannot differentiate functions with both an 'inout' parameter and a result
- Since differentiating multiple semantic results is pretty far away, I'd suggest just saying "cannot" instead of "cannot yet".

Reply:
Done: the new message is `cannot differentiate functions with both an 'inout' parameter and a result`.